src/core_plugins/kibana/public/home/components/tutorial/tutorial.js

```diff
@@ -104,6 +117,44 @@ export class Tutorial extends React.Component {
     });
   }
 
+  checkInstructionSetStatus = async (instructionSetIndex) => {
+    let isComplete = false;
+    let hasFailed  = false;
+    const instructions = this.getInstructions();
+    const esHitsCheckConfig = _.get(instructions, `instructionSets[${instructionSetIndex}].statusCheck.esHitsCheck`);
+    if (esHitsCheckConfig) {
+      const searchHeader = JSON.stringify({ index: esHitsCheckConfig.index });
+      const searchBody = JSON.stringify({ query: esHitsCheckConfig.query, size: 1 });
+      const body = `${searchHeader}\n${searchBody}\n`;
+
+      const response = await fetch(this.props.addBasePath('/elasticsearch/_msearch'), {
```

Comment:
I think _msearch is for when there are multiple queries and here there is only ever one, so you should be able to get away with a single _search, if you adjust your query a bit.  Not sure if it really makes any difference though, one way or the other. 🤷‍♀️  So take this feedback or leave it.

Reply:
Using `_search` requires putting the index name in the URL and I could not get it work that way. Not sure if its because how the call gets proxied by the Kibana server.